.github/workflows/axon-sync-test.yml

```diff
@@ -0,0 +1,82 @@
+name: Axon sync test
+
+on:
+  workflow_dispatch:
+
+jobs:
+  Build_and_Archive_Axon:
+    strategy:
+      matrix:
+        # Supported GitHub-hosted runners and hardware resources
+        # see https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
+        os: [ubuntu-22.04]
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
```

Comment:
should run on a self-hosted runner?

Reply:
The build can be run in different runners, and the final execution needs to be performed in the self-runner